content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -360,9 +363,9 @@ public class Main{
  
 ```
  
-In the main method, `Car` represents the interface class, and `Benz` represents the concrete class or child class. The child class gives its memory to the `Car` object to access its implemented methods. The `Benz` class implemented the start and stop behaviors of the `Car` class in the manner it wanted. This makes our application robust. We can change or add requirements to our application without breaking down the application.
+In the main method, *Car* represents the interface class, and *Benz* represents the concrete class or child class. The child class gives its memory to the *Car* object to access its implemented methods. The *Benz* class implemented the start and stop behaviors of the `Car` class in the manner it wanted. This makes our application robust. We can change or add requirements to our application without breaking down the application.
```

Comment:
Change the asterisks (/*) to backticks (/`) for consistency. Sometimes you use asterisks to denote the class name, other times backticks, do one or the other to stay consistent.